packages/manager/apps/dedicated/client/app/dedicated/server/installation/gabarit/dedicated-server-installation-gabarit.controller.js

```diff
@@ -165,20 +167,24 @@ angular
           validForm: true,
         };
 
-        Server.getHighestPriorityPartitionScheme(
+        Server.getPartitionSchemesByPriority(
           $stateParams.productId,
           $scope.installation.selectGabarit.id,
         )
+          .then((response) => {
+            $scope.installation.partitionSchemesList = response;
+            [$scope.installation.selectPartitionScheme] = response;
```

Comment:
This way the response parameter in the next then will be null and when you try accessing response[0], it will break.
You should return **response** instead of null

Reply:
```suggestion
            [$scope.installation.selectPartitionScheme] = response;
            return response;
```